src/os_auth/config.c

```diff
@@ -73,6 +75,13 @@ cJSON *getAuthdConfig(void) {
     if (config.manager_cert) cJSON_AddStringToObject(auth,"ssl_manager_cert",config.manager_cert);
     if (config.manager_key) cJSON_AddStringToObject(auth,"ssl_manager_key",config.manager_key);
 
+    if (config.force_options.enabled) cJSON_AddStringToObject(force, "enabled", "yes"); else cJSON_AddStringToObject(force, "enabled", "no");
+    if (config.force_options.key_mismatch) cJSON_AddStringToObject(force, "key_mismatch", "yes"); else cJSON_AddStringToObject(force, "key_mismatch", "no");
+    if (config.force_options.disconnected_time_enabled) cJSON_AddStringToObject(disconnected_time, "enabled", "yes"); else cJSON_AddStringToObject(disconnected_time, "enabled", "no");
+    if (config.force_options.disconnected_time) cJSON_AddNumberToObject(disconnected_time, "disconnected_time", config.force_options.disconnected_time);
```

Comment:
To avoid the duplication and nesting of **disconnected_time** I suggest renaming this to "value".
And the object will looks like:
`{"disconnected_time": {"enabled:"yes", "value":VALUE}}`

Reply:
It looks better, yes. Changed